### ISSUE TITLE
Some wrong tests

### ISSUE THREAD

**Issue by fluidsonic** [practicalswift/swift-compiler-crashes#61]
```
Apple Swift version 1.2 (swiftlang-602.0.37.3 clang-602.0.37)
```

Erroneously marked as crash (fuzzing) but works fine with Xcode 6.3 beta 1:
* 04592 swift constraints constraintsystem simplifyconstraint

Erroneously marked as fixed but still crashing with Xcode 6.3 beta 1 (also highlighted as dupes):
* 03396 swift typechecker conformstoprotocol
* 19198 swift typebase getanyoptionalobjecttype
* 22249 swift astcontext getspecializedconformance

hth :)

**Comment by practicalswift** [comment]
Thanks!

I've now corrected 03396.

However, for the other tests I'm getting opposite results:

```
$ xcrun swiftc --version
Apple Swift version 1.2 (swiftlang-602.0.37.3 clang-602.0.37)
Target: x86_64-apple-darwin14.1.0
$ xcrun swiftc crashes-fuzzing/04592-swift-constraints-constraintsystem-simplifyconstraint.swift 2>&1 | egrep Seg
<unknown>:0: error: unable to execute command: Segmentation fault: 11
$ xcrun swiftc fixed/19198-swift-typebase-getanyoptionalobjecttype.swift 2>&1 | egrep Seg
$ xcrun swiftc fixed/22249-swift-astcontext-getspecializedconformance.swift 2>&1 | egrep Seg
```

Anyone else who can chime in as a testing tiebreaker? :-)

**Comment by beltex** [comment]
Sure thing! I get the same exact results as you @practicalswift.

Interestingly, now in **Xcode 6.3 Beta 2** (`Apple Swift version 1.2 (swiftlang-602.0.42.4 clang-602.0.42`), 22249 segfaults in addition to 04592.
